sql/moz-fx-data-marketing-prod/acoustic/raw_recipient_raw_v1/query.sql

```diff
@@ -2,4 +2,6 @@ SELECT
   * EXCEPT (_file, _line, _fivetran_synced, _modified),
   DATE(@submission_date) AS submission_date,
 FROM
-  `moz-fx-data-bq-fivetran.acoustic_sftp.raw_recipient_export_v1`
+  `moz-fx-data-bq-fivetran.acoustic_sftp.raw_recipient_export_v_1`
+WHERE
+  DATE(@submission_date) = PARSE_DATETIME("%m/%d/%Y %H:%M:%S", event_timestamp)
```

Comment:
```suggestion
  DATE(@submission_date) = PARSE_DATETIME("%m/%d/%Y %H:%M:%S", event_timestamp)
```

It looks like we're comparing a DATE to a DATETIME here. Is that what we want? Can we wrap the datetime with `DATE` to make this more explicit?